dipy/reconst/tests/test_mcsd.py

```diff
@@ -58,42 +91,235 @@ def test_mcsd_model_delta():
 
 
 @pytest.mark.skipif(not mcsd.have_cvxpy, reason="Requires CVXPY")
```

Comment:
There is a PR #2188 related to mapmri and CVXPY, which is one of the workflow test that are failing, maybe it is related.
(Just so you know, maybe it is not your fault)